Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -126,6 +126,18 @@ theorem hasProd_ite_div_hasProd [DecidableEq β] (hf : HasProd f a) (b : β) :
     rw [Function.update_apply]
   · rw [div_mul_eq_mul_div, one_mul]
 
+/-- A more general version of `Multipliable.congr`. -/
+@[to_additive "A more general version of `Summable.congr`"]
+theorem Multipliable.congr' (hf : Multipliable f) (hfg : ∀ᶠ b in cofinite, f b = g b) :
```

Comment:
```suggestion
theorem Multipliable.congr_ae (hf : Multipliable f) (hfg : f =ᶠ[cofinite] g) :
```
Same below. Also, please add `congr_atTop` versions that assume that the index type is `Nat` and use `atTop` instead of `cofinite`.